### PR TITLE
Add new GH job to generate XLSX table and HTML page with SRG mapping

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Add new GH job to generate XLSX table and HTML page with SRG mapping

### CODE REVIEW

File: utils/create_srg_export.py [hunk 1]
```diff
@@ -249,6 +249,11 @@ def main() -> None:
                            used_rules)
         print(f"File written to {full_output}")
 
+    # export CSV to HTML table
+    import pandas as pd
```
Comment (Mab879) [inline utils/create_srg_export.py:253]: Do we want to have a switch for this?

**Review by Mab879 — COMMENTED** [review]
I think this is a good idea and will be helpful in the RHEL9 STIG development.

File: utils/create_srg_export.py (hunk 1 above)
Comment (ggbecker) [reply utils/create_srg_export.py:253]: Yes, we should definitely have an option for it. Also the script is in very early stages, it was just to have a proof of concept

Comment (ggbecker) [reply utils/create_srg_export.py]: updated

**Comment by ggbecker** [comment]
@Mab879 FYI

File: utils/create_srg_export.py [hunk 2]
```diff
@@ -36,6 +38,55 @@
...
+    'IA Control', 'CCI', 'SRGID', 'STIGID', 'SRG Requirement', 'Requirement',
+    'SRG VulDiscussion', 'Vul Discussion', 'Status', 'SRG Check', 'Check', 'SRG Fix',
+    'Fix', 'Severity', 'Mitigation', 'Artifact Description', 'Status Justification'
+    ]
+
+COLUMNS = string.ascii_uppercase[:17]  # A-Q uppercase letters
+
+COLUMN_MAPPINGS = dict(zip(COLUMNS, HEADERS))
+
+HTML_OUTPUT_TEMPLATE = '''
```
[the remark this answers is not in the document]
Comment (ggbecker) [reply utils/create_srg_export.py]: I've created a separate file to handle the HTML export

**Review by Mab879 — APPROVED** [review]

File: utils/create_srg_export.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mab879) [reply utils/create_srg_export.py]: That works. I was thinking about putting the template in another file but this works as well.

**Comment by ggbecker** [comment]
I haven't tested the files publishing as I can't do it without actually pushing to master, so there might be needed some extra rounds of commits to fix any issue that may appear in the gh-pages deployment

**Pull request merged**